index.bs

```diff
@@ -396,3 +422,81 @@ Acknowledgements {#ack}
 
 Cure53's [[DOMPURIFY]] is a clear inspiration for the API this document
 describes, as is Internet Explorer's {{window.toStaticHTML()}}.
+
+
+
+Appendix A: Interactions between Sanitizer And Trusted Types {#tt}
+======================================================
+
+<em>This section is not normative.</em>
+
+[[TrustedTypes]] can interact in helpful ways with the Sanitizer API. How to
+best do so will likely depend on the intended use of both. Because the
+Sanitizer API is meant to remove all scripting content, the default is to not
+consider the Sanitizer Trusted Types sensitive.
+
+The predominant use case would be to sanitize strings and use the resulting
+fragments directly, just as without Trusted Types:
+
+```js
+const untrusted_input = "....";
+some_node.AppendChildren(sanitizer.santize(untrusted_input));
+```
+
+For cases where handling of a string is desired, a sanitizing policy can easily
+be created using the `bound` method.
+
+```js
+const my_config = { ... };
+const sanitizing_policy = trustedTypes.createPolicy("apolicyname", {
+  createHTML: new Sanitizer(my_config).bound() });
```

Comment:
Do you want to refer to the arguments implicitly? 
E.g. createHTML(input) {return new Sanitizer(my_config}.bound(input) }. Policies createXYZ functions have variadic arguments after the first one, and with mentioning the argument explicitly we don't have to tie both TT and Sanitizer bound() signature (and are free to, for example, add arguments to `bound()` later).

Reply:
That was on purpose. Not sure if I'm being too clever here. :)

Minor misunderstanding: bound doesn't take parameters, but the function it retuns does. We do have the exact same problem there, though. E.g., if we were to redefine sanitizeToString to take two parameters that get concatenated, then my current definition would do weird things.

We could define bound() like so: return x => this.sanitizeToString(x);

IMHO, main 'contra' would be 1, that sanitizeToString doesn't really lend itself to extensions, so planning for such a case might not make that much sense, and 2, if it does change, we could change bound() to match. Main 'pro' is IMHO that... it's super easy to do, and additional clarity doesn't hurt.

I think I'll do an update for this case, by being more explicit about what bound() does.